board/models.py

```diff
@@ -1,3 +1,5 @@
 from django.db import models
-
-# Create your models here.
+ 
+#  #WARNING: NOT USED AS OF NOW
```

Comment:
Is this comment needed? if yes, pl give a one liner, else remove.

Reply:
ideally we should upload file related data in db. This comment says that we should do this later. (After we finalize db and its schema)